src/main/java/uk/gov/companieshouse/transactions/web/exception/ServiceException.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.transactions.web.exception;
+
+/**
+ * The class {@code ServiceException} is a form of {@code Exception}
+ * that should be used at the service layer to abstract lower level
+ * exceptions from being propagated up the call stack.
+ */
+public class ServiceException extends Exception {
+
+    /**
+     * Constructs a new {@code ServiceException} with a custom message and the specified
+     * cause.
+     *
+     * @param message a custom message
```

Comment:
separate params with something like `-` as it can be read wrong.

Reply:
Viewing the Javadoc in an IDE without the `-`, the formatting is correct. View the Javadoc in Intellij (ctrl + j) and the hyphen is present.

Reply:
Resolving this comment as was discussed off GitHub -  the `-` is automatically generated by JavaDoc itself, so inserting a `-` adds another one.